app/api/v1/endpoints/auth.py

```
@@ -0,0 +1,70 @@
+from app.infrastructure.persistence.repositories.user_repo import UserRepository
+from app.infrastructure.security.jwt import create_access_token
+from app.schemas.auth import LoginRequest, TokenResponse
+from app.schemas.user import UserResponse
+
+router = APIRouter()
+
+
+@router.post("/login", response_model=TokenResponse)
+async def login(
+    body: LoginRequest,
+    user_repo: UserRepository = Depends(get_user_repo),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)